frontend/libs/studio-components/src/components/StudioTableRemotePagination/StudioTableRemotePagination.tsx

```diff
@@ -0,0 +1,120 @@
+import { Label, NativeSelect, Pagination, Table } from '@digdir/design-system-react';
+import React, { forwardRef, useId } from 'react';
+import classes from './StudioTableRemotePagination.module.css';
+
+type LabelSize = 'small' | 'medium' | 'xsmall';
+type LabelSizeKeys = 'small' | 'medium' | 'large';
+export const labelSizeMap: Record<LabelSizeKeys, LabelSize> = {
+  small: 'xsmall',
+  medium: 'small',
+  large: 'medium',
+};
+
+export type Rows = (Record<string, React.ReactNode> & Record<'id', string | number>)[];
+
+export type PaginationProps = {
+  currentPage: number;
+  totalPages: number;
+  pageSizeOptions: number[];
+  pageSizeLabel: string;
+  onPageChange: (currentPage: number) => void;
+  onPageSizeChange: (currentSize: number) => void;
+  nextButtonText: string;
+  previousButtonText: string;
+  itemLabel: (num: number) => string;
+};
+
+type StudioTableRemotePaginationProps = {
+  columns: Record<'accessor' | 'value', string>[];
```

Comment:
Should we add a type for Columns?

```suggestion
   type Column = Record<'accessor' | 'value', string>;
   
  columns: Column[];
```